gTea/app/src/main/java/com/nechaieva/gtea/utils/LevenshteinDistCalculator.java

```diff
@@ -1,12 +1,45 @@
 package com.nechaieva.gtea.utils;
 
 public class LevenshteinDistCalculator {
```

Comment:
If you only have static methods here, did you consider to have a private constructor to avoid instanciations?